testsuite/capabilities.py

```diff
@@ -10,18 +10,34 @@
 @functools.cache
 def has_kuadrant():
     """Returns True, if Kuadrant deployment is present and should be used"""
-    spokes = weakget(settings)["control_plane"]["spokes"] % {}
-
-    for name, openshift in spokes.items():
-        # Try if Kuadrant is deployed
-        if not openshift.connected:
-            return False, f"Spoke {name} is not connected"
+    if "spokes" in weakget(settings)["control_plane"] % {}:
+        spokes = settings["control_plane"]["spokes"]
```

Comment:
^this will also trigger on `spokes: {}` in the settings
```suggestion
    if spokes := weakget(settings)["control_plane"]["spokes"] % {}:
```